components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -4973,6 +5007,52 @@ definitions:
         description: To determine whether analytics is enabled or not
         example: false
 
+  #-----------------------------------------------------
+  # The Scope Role Mapping List resource
+  #-----------------------------------------------------
+  ScopeList:
+    title: Scope Role Mapping List
+    properties:
+      count:
+        type: integer
+        description: |
+          Number of scopes available for tenant.
+        example: 60
+      list:
+        type: array
+        items:
+          $ref: '#/definitions/Scope'
+
+  #-----------------------------------------------------
+  # The Scope Role resource
```

Comment:
Why do we named it as Scope Role resource? I think we can change it as just ```The Scope resource```. WDYT?

Reply:
corrected: dc906c5